### ISSUE TITLE
FIX: neuronpedia_runner and tutorial notebook

### ISSUE THREAD

**Issue by hijohnnylin** [jbloomAus/SAELens#73]
It's broken after some recent refactoring.

- `correlated_neurons_l1` and `frac_nonzero` seem to be missing or relocated
- should reduce redundancy in outputs: logits bg values, freq_bar values
